### DISCUSSION TITLE
dct:provenance

### DISCUSSION THREAD

**Discussion by briesenberg07** [uwlib-cams/uwlswd#16] [modeling]
@gerontakos suggests (edited slightly):

> [...] describe the history of our ['MARC 008' RDF datasets](https://github.com/uwlib-cams/uwlswd_vocabs_marc_006_008) using a triple in the RDF data. We are going to state:

```
@prefix dct: <http://purl.org/dc/terms/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .

<term> dct:provenance <http://uwlswd_iri_here> .

[...]

<http://uwlswd_iri_here> rdf:value "provenance statement here"@en .
```

> This is similar to the way that we handled [Instances of the RightsStatement Class](https://doi.org/10.6069/uwlib.55.A.3.5) when publishing digital-collections metadata as LOD. If we do this, it would logically be stored in [uwlswd_datasets_dpla](https://urldefense.com/v3/__https:/github.com/uwlib-cams/uwlswd_datasets_dpla/tree/main__;!!K-Hz7m0Vt54!h2g5BvSrtrdcE-mGkVGvYPc9X7XMF4zkF4gmwButRJRADY9q85gotM4mSFteJKeTR5InKo_bO50$).  
> Were we to go forward with that, we should consider just calling that repo "datasets" and not "datasets_dpla." Other alternatives: (1) create a new repo for prov statements and other orphan datasets; (2) store it in uwswd_metadata and expand the identity of that repo beyond DataCite XML metadata.

**Comment by briesenberg07** [comment]
Hello @gerontakos and @dkreisstomkins \: Thanks for making the proposal Theo! Seems reasonable to me. 

**My question:** What would the new resources having `rdf:value "provenance statement"@en` be classed as?

**Reply by briesenberg07** [reply]
Also I see no significant problems with renaming/reorganizing [uwlswd_datasets_dpla](https://github.com/uwlib-cams/uwlswd_datasets_dpla) as needed to acommodate the new resource(s).

**Reply by gerontakos** [reply]
Classed as http://purl.org/dc/terms/ProvenanceStatement. However, if other properties apply, like PROV properties, we would want to also class as, say, prov:Entity. No specific plans to do that exist at present.

**Reply by briesenberg07** [reply]
RE: Where to store the new triples, do we create a ProvenanceStatement dataset? (If we wanted to follow existing naming conventions we might call it "Instances of the ProvenanceStatement Class" but we don't have to!)

**Reply by gerontakos** [reply]
Yes, I was thinking Instances of the ProvenanceStatement Class" stored alongside "Instances of the RightsStatement Class" in the repo "uwlswd_datasets_dpla" renamed "uwlswd_datasets." But we don't have to!
